src/main/java/mate/academy/internet/shop/dao/jdbc/JdbcOrderDao.java

```diff
@@ -8,13 +8,13 @@
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Optional;
-import mate.academy.internetshop.dao.OrderDao;
-import mate.academy.internetshop.dao.ProductDao;
-import mate.academy.internetshop.lib.Dao;
-import mate.academy.internetshop.lib.Inject;
-import mate.academy.internetshop.model.Order;
-import mate.academy.internetshop.model.Product;
-import mate.academy.internetshop.util.ConnectionUtil;
+import mate.academy.internet.shop.dao.OrderDao;
+import mate.academy.internet.shop.dao.ProductDao;
+import mate.academy.internet.shop.lib.Dao;
+import mate.academy.internet.shop.lib.Inject;
+import mate.academy.internet.shop.model.Order;
+import mate.academy.internet.shop.model.Product;
+import mate.academy.internet.shop.util.ConnectionUtil;
 
 @Dao
 public class JdbcOrderDao implements OrderDao {
```

Comment:
let's change it to OrderDaoJdbc, it would be easier to find(I may not know that this Jdbc or  something else but I'm sure that I'm looking for OrderDao)